apps/full-site-editing/full-site-editing-plugin/lib/feature-flags/feature-flags.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Feature flags file.
+ *
+ * @package full-site-editing
+ */
+
+/**
+ * Class A8C_Full_Site_Editing_Feature_Flags
+ */
+class A8C_Full_Site_Editing_Feature_Flags {
+	/**
+	 * Class instance.
+	 *
+	 * @var A8C_Full_Site_Editing_Feature_Flags
+	 */
+	private static $instance = null;
+
+	/**
+	 * Feature flags.
+	 *
+	 * @var array
+	 */
+	private $flags = array();
+
+	/**
+	 * Parameter and cookie name for storing/retrieving the feature flags.
+	 *
+	 * @var string
+	 */
+	const FEATURE_FLAGS = 'fse_feature_flags';
+
+	/**
+	 * A8C_Full_Site_Editing_Feature_Flags constructor.
+	 */
+	private function __construct() {
+		$this->set_flags();
+	}
+
+	/**
+	 * Creates instance.
+	 *
+	 * @return A8C_Full_Site_Editing_Feature_Flags
+	 */
+	public static function get_instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Get the feature flags.
+	 *
+	 * @return array
+	 */
+	public function get_flags() {
+		return $this->flags;
+	}
+
+	/**
+	 * Check if a feature flag is enabled.
+	 *
+	 * @param  string $flagName Feature flag.
+	 * @return boolean
+	 */
+	public function is_enabled( $flagName ) {
```

Comment:
I must have done something wrong in `phpcs.xml`, camelCase variable names should trigger something.

Reply:
Updated in ca80013dca76a3546a24ecaf596636b074e25de7.